.gitignore

```diff
@@ -18,3 +18,5 @@ coverage.html
 build/*/tmp/*
 contrib/build/*/tmp/*
 .pkg
+.kube
+.var
```

Comment:
as stupid as it might sound, I'd like to see some comments in the ignore for why we're ignoring things. future issue maybe.